spec/factories/user.rb

```diff
@@ -17,5 +17,24 @@
       sequence(:display_name) { |n| "admin_#{n}_login" }
       sequence(:login) { |n| "admin_#{n}_login" }
     end
+
+    factory :unique_user do
```

Comment:
Took the chance to add a temporary solution to one of my minor annoyances with Factories when combined with `transactional_fixtures = false` specs.

The issue is, doing just `sequence(:display_name) { |n| "admin_#{n}_login" }` without resetting the database each time we run specs IS FAILING at times, due to `n` here not accounting for duplicates (since each specs run does not delete the created data necessarily)

Using SecureRandom should work here. Initially I planned to add this to the already defined factory bots above, but it caused some legacy specs (especially Devise specs which needed to query  created user by `user_#{index}_login`). So I decided we should add a sub-factory here :unique_user so it will always create a unique user regardless if we reset test db or not.